src/subdomains/core/buy-crypto/route/buy.controller.ts

```diff
@@ -124,11 +124,10 @@ export class BuyController {
     );
   }
 
-  async getFees(userId: number, asset: Asset): Promise<{ fee: number; refBonus: number }> {
-    if (asset?.category === AssetCategory.STOCK) return { fee: 0, refBonus: 0 };
+  async getFees(userId: number, asset: Asset): Promise<{ fee: number }> {
+    if (asset?.category === AssetCategory.STOCK) return { fee: 0 };
```

Comment:
Move this also into `userService. getUserBuyFee`, to have everything in one place?